BibBuddy/app/src/main/res/values/strings.xml

```diff
@@ -21,6 +21,12 @@
     <string name="back">zurück</string>
     <string name="add">+</string>
     <string name="invalid_name">Bitte einen gültigen Namen angeben. Reine Whitespaces sind nicht erlaubt.</string>
+    <string name="cancel">abbrechen</string>
+    <string name="delete">löschen</string>
```

Comment:
Since the Tabs "Export" and "Hilfe" are written with capital letters -> I suggest changing "löschen" to "Löschen".